app/src/main/java/org/p2p/wallet/alarmlogger/model/AlarmSendErrorConverter.kt

```diff
@@ -0,0 +1,177 @@
+            amount = amount,
+            currency = currencyMode.getCurrencyModeSymbol()
+        )
+
+        val fees = buildSendFee(feePayerToken, accountCreationFee, transactionFee)
+
+        val relayAccountState = AlarmErrorsSendRequest.RelayAccountState(
+            created = relayAccount.isCreated,
+            balance = relayAccount.balance?.toString() ?: "none",
+        )
+
+        val request = AlarmErrorsSendRequest(
+            tokenToSend = tokenToSend,
+            fees = fees,
+            relayAccountState = relayAccountState,
+            userPubkey = userPublicKey,
+            recipientPubkey = recipientAddress.addressState.address.toBase58Instance(),
+            recipientName = (recipientAddress as? SearchResult.UsernameFound)?.username.orEmpty(),
+            simulationError = error.getSimulationError(),
+            feeRelayerError = error.getFeeRelayerError(),
+            blockchainError = error.getBlockchainError()
+        )
+
+        return AlarmErrorsRequest(
+            logsTitle = "Send Android Alarm",
+            payload = gson.toJson(request)
+        )
+    }
+
+    private fun buildSendFee(
+        feePayerToken: Token.Active,
+        accountCreationFee: String?,
+        transactionFee: String?
+    ): AlarmErrorsSendRequest.Fees {
+        if (accountCreationFee == null && transactionFee == null) {
+            return AlarmErrorsSendRequest.Fees(
+                transactionFeeAmount = 0.toString(),
+                accountCreationFee = null
+            )
+        }
+
+        return AlarmErrorsSendRequest.Fees(
+            transactionFeeAmount = transactionFee ?: 0.toString(),
+            accountCreationFee = AlarmErrorsSendRequest.AccountCreationFee(
+                paymentToken = AlarmErrorsSendRequest.PaymentToken(
+                    name = feePayerToken.tokenName,
+                    mint = feePayerToken.mintAddress.toBase58Instance()
+                ),
+                amount = accountCreationFee ?: 0.toString()
+            )
+        )
+    }
+
+    fun toSendViaLinkError(
+        token: Token.Active,
+        lamports: BigInteger,
+        userPublicKey: Base58String,
+        currency: String,
+        error: Throwable
+    ): AlarmErrorsRequest {
+        val tokenToSend = AlarmErrorsSvlRequest.TokenToSend(
+            tokenName = token.tokenName,
+            mint = token.mintAddress.toBase58Instance(),
+            amount = lamports.fromLamports(token.decimals).toPlainString(),
+            currency = currency
+        )
+
+        val request = AlarmErrorsSvlRequest(
+            tokenToSend = tokenToSend,
+            tokenToClaim = null,
+            userPubkey = userPublicKey,
+            simulationError = error.getSimulationError(),
+            feeRelayerError = error.getFeeRelayerError(),
+            blockchainError = error.getBlockchainError(),
+        )
+
+        return AlarmErrorsRequest(
+            logsTitle = "Link Create Android Alarm",
```

Comment:
Could we pass version in the title? I think it will be more explicit and useful when alarm comes to discord

Reply:
There is a version in the body but I can add here as well, np

Reply:
oh yeah, thanks
i think we need notify QA about this too. 

and also, how about adding a flag that this alarm triggered while running autotests?